Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -11,16 +20,26 @@ namespace Fuse.Internal
 	struct MiniList<T> : IList<T> where T : class
 	{
 		object _list;
+		ObjectList<T> AsList { get { return (ObjectList<T>)_list; } }
+		
+		MiniListMode _mode = MiniListMode.Empty;
```

Comment:
I've added a comment to the class.
If you can think of a better name it'd be easy enough to change as it's not a public API.